screens/backupAndRestore/BackupAndRestoreScreen.tsx

```diff
@@ -142,7 +145,32 @@ const BackupAndRestoreScreen: React.FC<BackupAndRestoreProps> = props => {
         <Text
           testID="storageInfo"
           style={Theme.BackupAndRestoreStyles.backupProgressText}>
-          {t('storage')}
+          {t('storage', {
+            driveName: getDriveName(),
+            accountType: getAccountType(),
+          })}
+        </Text>
+      </View>
+      <AccountInformation
+        email={props.profileInfo?.email}
+        picture={props.profileInfo?.picture}
+      />
+    </SectionLayout>
+  );
+
```

Comment:
Can we merge AccoutSection & IOSAccountSection to ignore duplication,?

Reply:
merged